### ISSUE TITLE
Please add a Register-CohesityProtectionSource command for GCP.

### ISSUE THREAD

**Issue by KuJoe** [cohesity/cohesity-powershell-module#210]
## 🚀 Feature Request

Currently all GCP sources need to be added via the GUI which is time consuming if you manage a lot of projects in GCP. Please add the ability to add these sources via Powershell similar to VMware and AWS. Thank you.

**Comment by pyashish** [comment]
Thanks for the request @KuJoe , we are looking at this. The is adding GCP to as a backup target to cohesity cluster correct ?

**Comment by KuJoe** [comment]
Sorry for the confusion, I meant as a protection source similar to Register-CohesityProtectionSourceAWS and Register-CohesityProtectionSourceVMware

**Comment by pyashish** [comment]
Right, you still meant adding it to cluster endpoint and not our Helios SaaS service, correct ?

**Comment by KuJoe** [comment]
Correct
